raiden/messages.py

```diff
@@ -704,11 +742,18 @@ def from_event(cls, event: SendLockedTransfer) -> "LockedTransfer":
 
 @dataclass(repr=False, eq=False)
 class RefundTransfer(LockedTransfer):
-    """ A special LockedTransfer sent from a payee to a payer indicating that
-    no route is available, this transfer will effectively refund the payer the
-    transfer amount allowing him to try a new path to complete the transfer.
+    """ A message used when a payee does not have any available routes to
+    further the transfer.
```

Comment:
```suggestion
    forward the transfer.
```

Reply:
I had to search, didn't know `further` can actually be used as a verb too. But I also think `forward` would be clearer here